File: PRAK605-2310817310008-MUHAMMADRIZKIRAMADHAN.c
#include <stdio.h>

void perkalianmatriks(int ordo, int matriksA[ordo][ordo], int matriksB[ordo][ordo], int hasil[ordo][ordo]){
    for (int i = 0; i < ordo; i++){
        for (int j = 0; j < ordo; j++){
            hasil[i][j] = 0;
            for (int k = 0; k < ordo; k++){
                hasil[i][j] += matriksA[i][k] * matriksB[k][j];
            }
        }
        
    }
}
int main(){
    int ordo;
    printf(" ");
    scanf("%d", &ordo);

    int matriksA[ordo][ordo], matriksB[ordo][ordo], hasil[ordo][ordo];

    printf("Matriks A");
        for (int i = 0; i < ordo; i++){
            for (int j = 0; j < ordo; j++){
                scanf("%d", &matriksA[i][j]);
            }
        }
    printf("Matriks B");
        for (int i = 0; i < ordo; i++){
            for (int j = 0; j < ordo; j++){
                scanf("%d", &matriksB[i][j]);
            }
        }
        perkalianmatriks(ordo, matriksA, matriksB, hasil);

    printf("Matriks AxB\n");
        for (int i = 0; i < ordo; i++){
            for (int j = 0; j < ordo; j++){
                printf("%d ", hasil[i][j]);
            }
            printf("\n");
        }
    return 0;
}

File: PRAK605-2310817310008-MUHAMMADRIZKIRAMADHAN.py
def perkalianmatriks(ordo, matriksA, matriksB):
    hasil = [[0 for _ in range(ordo)] for _ in range(ordo)]
    for i in range(ordo):
        for j in range(ordo):
            hasil[i][j] = 0
            for k in range(ordo):
                hasil[i][j] += matriksA[i][k] * matriksB[k][j]
    return hasil

ordo = int(input(" "))
matriksA = []
print("Matriks A")
for i in range(ordo):
    baris = list(map(int, input().split()))
    matriksA.append(baris)
    
matriksB = []
print("Matriks B")
for i in range(ordo):
    baris = list(map(int, input().split()))
    matriksB.append(baris)

hasil = perkalianmatriks(ordo, matriksA, matriksB)
    
print("Matriks AXB")
for i in range(ordo):
    for j in range(ordo):
        print(hasil[i][j], end=" ")
    print()